kalamine/xkb_manager.py

```diff
@@ -304,6 +310,8 @@ def list_symbols(xkb_root: Path, kb_index: Dict) -> Dict:
         with open(path, "r", encoding="utf-8") as symbols:
             for line in symbols:
                 name = is_new_symbol_mark(line)
+                if name is None:
+                    continue
```

Comment:
It does not change the code behavior since None cannot be a key of `variants`. But it helps readers and type checkers to understand what is happening. Does this explanation make sense?